keps/sig-node/2625-cpumanager-policies-thread-placement/kep.yaml

```diff
@@ -1,43 +1,51 @@
 title: SMT aware cpumanager policy
 kep-number: 2625
 authors:
-  - "@fromanirh"
+  - "@ffromani"
   - "@swatisehgal"
 owning-sig: sig-node
 participating-sigs: []
-status: implementable
+status: implemented
```

Comment:
should this be changed after the GA graduation or can we change it in the context of the GA graduation?

Reply:
I have seen it done both ways and don't have a strong preference.  I think we can mark it `implemented` now itself in the context of GA graduation.

Reply:
I think the enhancement CI machinery probably ignores KEPs marked `implemented` - I would leave this `implementable` until after it actually makes it in the release.